client/src/contexts/AuthContext.tsx

```diff
@@ -40,33 +35,27 @@ export const AuthProvider = ({
         }
       })
       .catch((reason: AxiosError) => {
-        // Catch 401 which signals an unauthorized user, which is not an issue
-        if (!(reason.response && reason.response.status === 401)) {
+        // Catch 400 or 401 which signals an unauthorized user, which is not an issue
+        if (
+          !(
+            reason.response &&
```

Comment:
do we still need this truthiness check? and if so can we just do `reason.response?.status` below?

Reply:
Good catch